src/java/org/apache/cassandra/auth/CassandraRoleManager.java

```diff
@@ -732,4 +737,32 @@ public Supplier<Map<RoleResource, Set<Role>>> bulkLoader()
             return entries;
         };
     }
+
+    protected void scheduleDisconnectInvalidRoleTask()
+    {
+        // Delay between executions, with jitter
+        int period = (int) DatabaseDescriptor.getInvalidClientDisconnectPeriod(TimeUnit.MILLISECONDS);
```

Comment:
I see methods to set these properties (period+jitter), however if you statically retrieve the value once, setting them anywhere else will have no effect. So if the desire is to be able to have some mechanism to update these values while the process is running (i.e. JMX), we should probably retrieve these values dynamically as well.